setup.py

```diff
@@ -1,11 +1,28 @@
+import os
+import sys
+
 from setuptools import find_packages, setup
+from setuptools.command.install import install
 
 with open('VERSION') as version_fd:
     VERSION = version_fd.read().strip()
 
 with open('README.rst') as readme_fd:
     long_description = readme_fd.read()
 
+class VerifyVersionCommand(install):
+    """Custom command to verify that the git tag matches our version"""
+    description = 'verify that the git tag matches our version'
+
+    def run(self):
+        tag = os.getenv('CIRCLE_TAG')
```

Comment:
nit: should we bail out with a different message if `tag is None`?

Reply:
Aye.  This depends on the script being run on `circle` on a tagged branch. I can add some sanity.